src/components/new-test/QuestionBlock.tsx

```diff
@@ -5,15 +5,22 @@ import { SuperSelect } from './SuperSelect';
 import { SuperInput } from './SuperInput';
 import { Timer } from './Timer';
 
+
 type QuestionBlockType = {
   value: string;
+  difficultyValue: string;
```

Comment:
i think just 'difficulty' would be enough. an it should be Enum type

Reply:
fix naming and add enum type